.github/workflows/benchmarks.yml

```diff
@@ -13,20 +13,16 @@ jobs:
     name: Run benchmarks
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout
+      - name: Checkout branch
         uses: actions/checkout@v4
+      - name: Set up pixi
+        uses: prefix-dev/setup-pixi@v0.8.1
         with:
-          ref: main
-          fetch-depth: 0
-      - name: Setup conda
-        uses: mamba-org/setup-micromamba@v1
-        with:
-          condarc-file: .github/assets/.condarc
-          environment-file: benchmarks/environment.yml
-      - name: Install package
-        run: python -m pip install -e .
-      - name: Run benchmark.py
-        run: python benchmarks/benchmark.py
+          environments: default lint
```

Comment:
Is there a reason why we need the `lint` environment here?

Reply:
Great catch!
[8a43f13](https://github.com/Quantco/metalearners/pull/12/commits/8a43f13d397bded9b8dac8a926262bff5823a924)